subgraphs/dopex/schema.graphql

```diff
@@ -0,0 +1,1526 @@
+  dailyProtocolSideRevenueUSD: BigDecimal!
+
+  " Gross revenue for the protocol (revenue claimed by protocol). Examples: AMM protocol fee (Sushi’s 0.05%). OpenSea 10% sell fee. "
+  cumulativeProtocolSideRevenueUSD: BigDecimal!
+
+  " Daily revenue generated by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. "
+  dailyTotalRevenueUSD: BigDecimal!
+
+  " All revenue generated by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. "
+  cumulativeTotalRevenueUSD: BigDecimal!
+
+  " Daily premium paid to enter a position. "
+  dailyEntryPremiumUSD: BigDecimal!
+
+  " All premium paid to enter a position. "
+  cumulativeEntryPremiumUSD: BigDecimal!
+
+  " Daily premium paid to exit a position. "
+  dailyExitPremiumUSD: BigDecimal!
+
+  " All premium paid to exit a position. "
+  cumulativeExitPremiumUSD: BigDecimal!
+
+  " Daily all premium paid. "
+  dailyTotalPremiumUSD: BigDecimal!
+
+  " All historical premium paid. "
+  cumulativeTotalPremiumUSD: BigDecimal!
+
+  " Daily premium paid to deposit in liquidity pools. "
+  dailyDepositPremiumUSD: BigDecimal!
+
+  " All premium paid to deposit in liquidity pools. "
+  cumulativeDepositPremiumUSD: BigDecimal!
+
+  " Daily premium paid to withdraw from liquidity pools. "
+  dailyWithdrawPremiumUSD: BigDecimal!
+
+  " All premium paid to withdraw from liquidity pools. "
+  cumulativeWithdrawPremiumUSD: BigDecimal!
+
+  " Daily premium paid to enter and exit liquidity pool. "
+  dailyTotalLiquidityPremiumUSD: BigDecimal!
+
+  " All premium paid to enter and exit liquidity pool. "
+  cumulativeTotalLiquidityPremiumUSD: BigDecimal!
+
+  " Daily number of puts minted "
+  dailyPutsMintedCount: Int!
+
+  " Total number of puts minted "
+  putsMintedCount: Int!
+
+  " Daily number of calls minted "
+  dailyCallsMintedCount: Int!
+
+  " Total number of calls minted "
+  callsMintedCount: Int!
+
+  " Daily number of contracts minted "
+  dailyContractsMintedCount: Int!
+
+  " Total number of contracts minted "
+  contractsMintedCount: Int!
+
+  " Daily number of contracts taken "
+  dailyContractsTakenCount: Int!
+
+  " Total number of contracts taken "
+  contractsTakenCount: Int!
+
+  " Daily number of contracts expired "
+  dailyContractsExpiredCount: Int!
+
+  " Total number of contracts expired "
+  contractsExpiredCount: Int!
+
+  " Daily number of contracts exercised "
+  dailyContractsExercisedCount: Int!
+
+  " Total number of contracts exercised "
+  contractsExercisedCount: Int!
+
+  " Daily number of contracts exercised+expired "
+  dailyContractsClosedCount: Int!
+
+  " Total number of contracts exercised+expired "
+  contractsClosedCount: Int!
+
+  " Daily number of open positions (minted-closed positions) "
+  dailyOpenPositionCount: Int!
+
+  " Cumulative number of open positions (minted positions) "
+  _cumulativeOpenPositionCount: Int!
```

Comment:
Maybe this should be in the schema?

Reply:
created the PR at https://github.com/messari/subgraphs/pull/1923 and posted the message at https://discord.com/channels/953684103012683796/1067631209968058408/1088081279859302563